### DISCUSSION TITLE
undervolting cpu 5800H

### DISCUSSION THREAD

**Discussion by CIAO1987** [JamesCJ60/Universal-x86-Tuning-Utility#103] [Q&A]
Hi, is it possible to do undervolting on amd 5800h cpu?

**Comment by JamesCJ60** [comment]
As of the 5000 series, only Ryzen 9 mobile APUs have the ability to undervolt.

**Reply by CIAO1987** [reply]
I understand, so the only way to reduce the temperatures is to lower the temp limit or the tdp? do you know another trick to not reduce performance too much and keep temps low?

**Reply by Jeju2004** [reply]
I suggest that you disable your processor performance boost mode.
Directory: 
- Open your windows and then type "edit power plan" and run the software.
- Click the "change advanced power settings."
- Find processor power management and then click it.
- Once you find the processor performance boost mode tab, choose the "disabled" option form the toggle at the right side.

Note: Make sure that you are on your preferred mode when changing this setting, it will be regarded at the top of the 'power options window.
